### PR TITLE
Add sphinxcontrib-autoprogram

### PR DESCRIPTION
**Pull request by asmeurer**

### CODE REVIEW

File: recipes/sphinxcontrib-autoprogram/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    - sphinx >=1.2
+    - six
+
+test:
+  imports:
+    - sphinxcontrib
+
+about:
+  home: https://bitbucket.org/birkenfeld/sphinx-contrib
+  license: BSD License
```
Comment (jakirkham) [inline recipes/sphinxcontrib-autoprogram/meta.yaml]: Please specify clauses (e.g. `BSD 3-Clause`). Also please drop `License`.

Comment (jakirkham) [inline recipes/sphinxcontrib-autoprogram/meta.yaml]: Looks like [BSD 2-Clause]( https://bitbucket.org/birkenfeld/sphinx-contrib/src/21e3b2dc70a43d3f7e8942c4aaf078f66e1c575b/LICENSE?fileviewer=file-view-default ).

**Comment by jakirkham** [comment]
One minor thing. Otherwise LGTM.

**Comment by asmeurer** [comment]
Fixed.

**Comment by jakirkham** [comment]
Thanks @asmeurer.

**Comment by asmeurer** [comment]
Nice, from PR to built package in an hour.

**Pull request merged**